packages/cli-tool/src/commands/generate/index.ts

```diff
@@ -65,11 +77,39 @@ export default class Generate extends Command {
       }
 
       this.setVersionControl(appName, answers.versionControl)
+      await this.setUIFramework(appName, answers.uiFramework)
+
+      // Clean files after all steps
+      this.cleanFiles(appName)
     } catch (error) {
       this.error(error as string | Error)
     }
   }
 
+  setUIFramework = async(
+    appName: string,
+    uiFramework: string,
+  ): Promise<void> => {
+    if (uiFramework === 'bootstrap') {
+      cli.info('Configure Bootstrap...')
+      const result = await this.config.runHook('install-bootstrap', {
+        appName: appName,
+      })
+
+      if (hookFailed(result)) {
+        const errorMsg = formatHookErrorMsg(result)
+        cli.info(
+          'Something went wrong while setting up bootstrap...',
+          errorMsg,
+        )
+      }
```

Comment:
How about moving this to a function `setupBootstrap()` so the `setUIFramework` function looks a bit cleaner? 💡

Reply:
In discussion [here](https://github.com/nimblehq/react-templates/pull/107#discussion_r915508945) 🙇

Reply:
Done in 8f7cd6d :)